.azuredevops/pipelineTemplates/jobs.validateModuleDeployment.yml

```diff
@@ -359,6 +359,7 @@ jobs:
                       DeploymentName    = '$(deploymentName)'
                       TemplateFilePath  = $templateFilePath
                       ResourceGroupName = '${{ parameters.resourceGroupName }}'
+                      subscriptionId    = '${{ parameters.subscriptionId }}'
```

Comment:
I guess the same should be added to the GitHub action `validateModuleDeployment` too, right?

Link: https://github.com/Azure/ResourceModules/blob/main/.github/actions/templates/validateModuleDeployment/action.yml#:~:text=Initialize%2DDeploymentRemoval.ps1%27)-,if%20(%2Dnot%20%5BString%5D%3A%3AIsNullOrEmpty(%27%24%7B%7B%20steps.deploy_step.outputs.deploymentName%20%7D%7D%27))%20%7B,%7D,-Write%2DVerbose%20%27Invoke

Reply:
As I know in the github Action, there is a explicit Login to the given Subscription. In ADO we just use the ServiceConnection which can be assigned to other Subscriptions as well.

Reply:
Ah right.